phpunit/functional/Item_SoftwareLicenseTest.php

```diff
@@ -107,80 +105,74 @@ public function testGetLicenseForInstallation()
 
         //simulate license install
         $lic = getItemByTypeName('SoftwareLicense', '_test_softlic_1');
-        $this->assertTrue(
-            $lic->update([
-                'id'                       => $lic->fields['id'],
-                'softwareversions_id_use'  => $version1->fields['id'],
-            ])
-        );
+        $this->updateItem('SoftwareLicense', $lic->fields['id'], [
+            'softwareversions_id_use' => $version1->fields['id'],
+        ]);
 
         $this->assertCount(
             1,
-            \Item_SoftwareLicense::getLicenseForInstallation(
+            Item_SoftwareLicense::getLicenseForInstallation(
                 'Computer',
                 $computer1->fields['id'],
                 $version1->fields['id']
             )
         );
 
         //reset license
-        $this->assertTrue(
-            $lic->update([
-                'id'                       => $lic->fields['id'],
-                'softwareversions_id_use'  => 0,
-            ])
-        );
+        $this->updateItem('SoftwareLicense', $lic->fields['id'], [
+            'softwareversions_id_use' => 0,
+        ]);
     }
 
     public function testAddUpdateDelete()
     {
         $computer1 = getItemByTypeName('Computer', '_test_pc01');
         $computer2 = getItemByTypeName('Computer', '_test_pc02');
         $computer3 = getItemByTypeName('Computer', '_test_pc11');
-        $lic       = getItemByTypeName('SoftwareLicense', '_test_softlic_4');
+        $lic = getItemByTypeName('SoftwareLicense', '_test_softlic_4');
 
         // Do some installations
-        $lic_computer = new \Item_SoftwareLicense();
+        $lic_computer = new Item_SoftwareLicense();
 
         $input = [
-            'items_id'              => $computer1->fields['id'],
-            'itemtype'              => 'Computer',
-            'softwarelicenses_id'   => $lic->fields['id'],
+            'items_id' => $computer1->fields['id'],
+            'itemtype' => 'Computer',
+            'softwarelicenses_id' => $lic->fields['id'],
         ];
-        $this->assertGreaterThan(0, (int) $lic_computer->add($input));
+        $this->createItem(Item_SoftwareLicense::class, $input)->getID();
```

Comment:
The getID() is not needed here
```suggestion
        $this->createItem(Item_SoftwareLicense::class, $input);
```